src/SymOp.jl

```diff
@@ -17,10 +17,9 @@ const SYMMETRY_TOLERANCE = 1e-5
 # Represents a symmetry (S,τ)
 struct SymOp
     S::Mat3{Int}
-    τ::Vec3{Float64}
+    τ::Vec3{Float64}  # floating-point type fixed by spglib
     function SymOp(S, τ)
-        τ = τ .- floor.(τ)
```

Comment:
Also is this not important to "normalise" the τ to [0, 1) (or at least roughtly). I think we need this at places.

Reply:
stupid me. That's why there is the mod.